certified-connectors/Gratavid/apiDefinition.swagger.json

```diff
@@ -0,0 +1,542 @@
+            "contact_firstName": {
+              "type": "string",
+              "description": "contact_firstName",
+              "title": "Contact First Name"
+            },
+            "contact_lastName": {
+              "type": "string",
+              "description": "contact_lastName",
+              "title": "Contact Last Name"
+            },
+            "replyText": {
+              "type": "string",
+              "description": "replyText",
+              "title": "Reply Message"
+            },
+            "replyVideoSrc": {
+              "type": "string",
+              "description": "replyVideoSrc",
+              "title": "Video Reply Src"
+            }
+          }
+        }
+      },
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "status": {
+                  "type": "string",
+                  "description": "status"
+                },
+                "data": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "string",
+                        "description": "id"
+                      },
+                      "label": {
+                        "type": "string",
+                        "description": "label",
+                        "title": "",
+                        "x-ms-visibility": "important"
+                      },
+                      "sample": {
+                        "type": "string",
+                        "description": "sample"
+                      },
+                      "value": {
+                        "type": "string",
+                        "description": "value"
+                      }
+                    }
+                  },
+                  "description": "data"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Gratavid Data",
+        "description": "Get Gratavid notes and users",
+        "operationId": "getData",
+        "parameters": [
+          {
+            "name": "source",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "default": "microsoftFlow",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "event",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "x-ms-visibility": "important",
+            "enum": [
+              "getUsers",
+              "getNotes"
+            ]
+          }
+        ],
+        "x-ms-visibility": "internal"
+      },
+      "delete": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Webhook Unsubscribe",
```

Comment:
@Wtrapp A single operation can not have two summary values. If there are more than one summary values in the operation, please remove them. And summary like this can be improved as these are visible to public.

Ex.: `"summary": "Unsubscribe the webhook"`

Reply:
The description says "Unsubscribe the webhook" now, should I make the summary say that too? Note: this is an internal event that is not visible to the public.